Dockerfile

```diff
@@ -11,16 +11,24 @@ CMD git ls-files --cached -z -- '*.rst' \
     | xargs -0 -- python3 -m sphinxlint --enable all --disable trailing-whitespace --max-line-length 2000
 
 FROM php:${PHP_VERSION}-cli-alpine${ALPINE_VERSION} as base
+
+RUN curl --location --output /usr/local/bin/install-php-extensions https://github.com/mlocati/docker-php-extension-installer/releases/latest/download/install-php-extensions \
```

Comment:
https://github.com/mvorisek/image-php/blob/a38b8a1ac/data/7.4-zts-alpine/Dockerfile#L14 looks better to me although the size will be a little larger

Reply:
Does this produce the same output, though? I mean, will cache be invalidated the same way for `curl` and `COPY --from`? I would rather not mix layers' cleanup with "logic" (binary's source) change, but I am not against this change in general.

Reply:
I will be managed by Docker - when a new version is released, it will COPY a new version and not reuse the following cached layers. IMO this is good and the mantra of Docker ;-)

Reply:
Yeah I know how it works when it comes to `COPY --from` and layers in general, I am just wondering if it can determine new versions from `RUN curl` the same way 🙂. Because if `COPY --from` approach would invalidate more often that would mean it's a behaviour change and I don't want to mix it with changes related to `pcntl` (even if more often invalidation on new releases is good).